env/dev/env/ios/main.cljs

```diff
@@ -20,7 +20,9 @@
 
 (figwheel/start {:websocket-url    (:ios conf/figwheel-urls)
                  :heads-up-display false
-                 :jsload-callback  #(swap! cnt inc)})
+                 :jsload-callback  (fn []
+                                       (println :CHANGES_LOADED)
```

Comment:
debug statement?

Reply:
yeah, i will remove it though i found it really useful sometimes

Reply:
maybe make a `(log/debug)` entry then?

Reply:
Don't forget to remove it or turn it into a debug statement